docs/server/v20/server/introduction/clients.md

```diff
@@ -25,29 +32,36 @@ EventStoreDB offers a low-level protocol in the form of an asynchronous TCP prot
 We plan to phase out the TCP protocol in the later versions. Please consider migrating your applications that use the TCP protocol and refactor them to use gRPC instead.
 :::
 
+Find out more about configuring the TCO protocol on the [TCP configuration](../networking/tcp.md) page.
+
 ### EventStoreDB supported clients
 
 - [.NET Framework and .NET Core](http://www.nuget.org/packages/EventStore.Client)
 - [JVM Client](https://github.com/EventStore/EventStore.JVM)
+- [Haskell](https://github.com/EventStore/EventStoreDB-Client-Haskell)
 
 ### Community developed clients
 
 - [Node.js](https://www.npmjs.com/package/event-store-client)
 - [Node.js](https://www.npmjs.com/package/ges-client)
 - [Node.js](https://github.com/nicdex/eventstore-node)
-- [Haskell](https://github.com/YoEight/eventstore)
 - [Erlang](https://bitbucket.org/anakryiko/erles)
 - [F#](https://github.com/haf/EventStore.Client.FSharp)
 - [Elixir](https://github.com/exponentially/extreme)
 - [Java 8](https://github.com/msemys/esjc)
 - [Maven plugin](https://github.com/fuinorg/event-store-maven-plugin)
-- [Rust](https://github.com/YoEight/eventstore-rs)
 - [Go](https://github.com/jdextraze/go-gesclient)
 - [PHP](https://github.com/prooph/event-store-client/)
 
 ## HTTP
 
-EventStoreDB also offers an HTTP-based interface, based specifically on the [AtomPub protocol](http://tools.ietf.org/html/rfc5023). As it operates over HTTP, this is less efficient, but nearly every environment supports it.
+EventStoreDB also offers an HTTP-based interface, based specifically on the [AtomPub protocol](http://tools.ietf.org/html/rfc5023). As it operates over HTTP, this is less efficient, but nearly every environment supports it. 
+
+Find out more about configuring the HTTP protocol on the [HTTP configuration](../networking/http.md) page.
+
+::: warning
+We plan to phase out the HTTP protocol in the later versions. Please consider migrating your applications that use the TCP protocol and refactor them to use gRPC instead.
```

Comment:
Its not that we are going to phase it out. We are probably going to replace it with something different, maybe HAL. This will likely sit over the top of the grpc

Reply:
Ok, then I'll remove this warning.